components/application-mgt/org.wso2.carbon.identity.application.mgt/pom.xml

```diff
@@ -273,7 +273,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Since the parent pom of the repo has the plugin enabled, let's remove the config from here.